Apps/WebClient/src/NewClientApp/src/views/errors/AppErrorView.vue

```diff
@@ -0,0 +1,49 @@
+<script setup lang="ts">
+import { computed } from "vue";
+
+import { AppErrorType } from "@/constants/errorType";
+import { useAppStore } from "@/stores/app";
+
+const appStore = useAppStore();
+
+const isTooManyRequests = computed<boolean>(() => {
+    return appStore.appError === AppErrorType.TooManyRequests;
+});
+</script>
+
+<template>
+    <v-layout>
+        <v-app-bar
+            class="border-b-md border-accent border-opacity-100"
+            color="primary"
+        >
+            <v-img
+                alt="Go to Health Gateway home page"
+                src="@/assets/images/gov/hg-logo-rev.svg"
+                max-width="143px"
+                class="pa-1"
+            />
```

Comment:
Suggest making this a link that refreshes the page.
```suggestion
            <a href="/">
                <v-img
                    alt="Go to Health Gateway home page"
                    src="@/assets/images/gov/hg-logo-rev.svg"
                    max-width="143px"
                    class="pa-1"
                />
            </a>
```